lib/runner/extract-runnable-items.js

```diff
@@ -186,6 +186,11 @@ var sdk = require('postman-collection'),
 
         items = findItemsOrGroups(collection, entrypointLookup, true);
 
+        // Sorting the items by the order in which they appear as folder arguments.
```

Comment:
This comment is misleading,  In the previous step, we queried to get both items and groups. So in this step, we are not just sorting by folders aka groups, but also requests aka items.